enclave/src/polkadex_balance_storage.rs

```diff
@@ -0,0 +1,189 @@
+use codec::{Decode, Encode};
+use log::*;
+use polkadex_sgx_primitives::{AccountId, AssetId, PolkadexAccount};
+use sgx_tstd::collections::HashMap;
+use sgx_tstd::hash::Hash;
+use sgx_tstd::hash::Hasher;
+use sgx_tstd::vec::Vec;
+use sgx_types::{sgx_epid_group_id_t, sgx_status_t, sgx_target_info_t, SgxResult};
+use sp_core::blake2_256;
+use std::sync::{
+    atomic::{AtomicPtr, Ordering},
+    Arc, SgxMutex, SgxMutexGuard,
+};
+
+static GLOBAL_POLKADEX_BALANCE_STORAGE: AtomicPtr<()> = AtomicPtr::new(0 as *mut ());
+
+pub type EncodedKey = Vec<u8>;
+
+#[derive(Encode, Decode, Clone, Debug, PartialEq, Eq)]
+pub struct Balances {
+    pub free: u128,
```

Comment:
Not really important, but if it's not much effort: It would be nice to have a type BalancesType = u128 (or something like this) defined instead of a u128

Reply:
I will do this after this is merged to develop. Created a new issue for it